index.html

```diff
@@ -0,0 +1,111 @@
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!--this is for the future style sheet-->
+    <link rel="stylesheet" type="text/css" href="styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>ADOPTADOG</h1>
+        <img src="adoptadog-logo.svg" width="500" alt="white dog silhoutte">
+        <h2>Seattle's Premier Dog Adoption Agency</h2>
+        <ing src="cart-icon.svg" width="200" alt="cart icon">
+          <a href="dog-checkout-page4.html">Checkout</a>
+      </header>
+      <nav>
+        <ul>
+          <li>Dogs</li>
+          <li>Blog</li>
+          <li>Adopt</li>
+        </ul>
+      </nav>
+      <div>
+        <img src="home-intro.jpg" width="500" alt="a boy hugs a happy Golden Retriver">
+          <h2> Adopt a dog. </h2>
+            <p>Puppy kitty ipsum dolor sit good dog foot stick canary. Teeth Mittens grooming vaccine walk swimming nest good boy furry tongue heel furry treats fish. Cage run fast kitten dinnertime ball run foot park fleas throw house train licks stick dinnertime window. Yawn litter fish yawn toy pet gate throw Buddy kitty wag tail ball groom crate ferret heel wet nose Rover toys pet supplies. Bird Food treats tongue lick teeth ferret litter box slobbery litter box crate bird small animals yawn small animals shake slobber gimme five toys polydactyl meow.
+            </p>
+        </div>
+        <div>
+          <img src="murphy-card.jpg" width="200" alt="picture of a terrier">
+            <h3>Murphy</h3>
+             <h4>cost to adopt $123.00</h4>
+               <p>Pet Food pet supplies gimme five puppy cage food feathers food heel feathers running pet gate walk lazy dog Spike. Good Boy park lazy dog walk kibble Scooby snacks licks canary.
+               </p>
+               <button type="button" onclick="alert('Say, Hello, to your new dog!')">Adopt</button>
+         </div>
+         <div>
+           <img src="poppy-card.jpg" width="200" alt="picture of a small fluffy white mutt">
+             <h3>Poppy</h3>
+              <h4>cost to adopt </h4>
+                <p>Pet Food pet supplies gimme five puppy cage food feathers food heel feathers running pet gate walk lazy dog Spike. Good Boy park lazy dog walk kibble Scooby snacks licks canary.
+                </p>
+                <button type="button" onclick="alert('Say, Hello, to your new dog!')">Adopt</button>
+          </div>
+          <div>
+            <img src="jack-card.jpg" width="200" alt="picture of a beagel">
+              <h3>Jack</h3>
+               <h4>cost to adopt $193.00</h4>
+                 <p>Pet Food pet supplies gimme five puppy cage food feathers food heel feathers running pet gate walk lazy dog Spike. Good Boy park lazy dog walk kibble Scooby snacks licks canary.
+                 </p>
+                 <button type="button" onclick="alert('Say, Hello, to your new dog!')">Adopt</button>
+           </div>
+           <div>
+             <img src="duffy-card.jpg" width="200" alt="picture of a Pitbull">
```

Comment:
for alt tags you don't need to say "picture of..." or "image of...." the fact that it's an image is implied. Just as long as the tag describes the image.